### ISSUE TITLE
Not downloading

### ISSUE THREAD

**Issue by israelmartins1** [puemos/hls-downloader#219]
The software used to download this kind of video whenever I want in the same conditions showed on the print. But now is not working. It is not downloading anything

![image](https://user-images.githubusercontent.com/60886450/144769857-c8f9066c-70c5-4123-9dc9-eb87e04d146d.png)

See after some minutes - still downloading anything.

![image](https://user-images.githubusercontent.com/60886450/144769946-9fb5a82f-0bfb-47ad-9b93-3ea9ab2641fc.png)

Video link: https://certifiquei-cnpi.club.hotmart.com/t/page/g97Ba80Lep

or

https://contentplayer.hotmart.com/video/oARKgoEmRb/source/playlist/dmlkZW8vb0FSS2dvRW1SYi9obHMvNjAwLzYwMC5tM3U4.m3u8

Browser Firefox 94.0.2 (64-bits)
My internet is really ok

**Comment by israelmartins1** [comment]
I've tried in another browser
![image](https://user-images.githubusercontent.com/60886450/144770657-7b33b074-b946-4171-9342-4e4d19697d2a.png)


After 3 minutes nothing was downloaded

![image](https://user-images.githubusercontent.com/60886450/144770775-afdc2d99-a40b-40b9-8057-d19ea7e602a6.png)

**Comment by hugolosker** [comment]
Yeah, same issue here on hotmart website! But on nutror website works fine!

![edge hotmart](https://user-images.githubusercontent.com/95626120/144847492-7207ce1d-97ed-405e-8047-cb0a7efbe9fe.jpg)
![edge nutror](https://user-images.githubusercontent.com/95626120/144847496-4264578f-3b76-420b-8d94-3eae814b76be.jpg)

**Comment by PsxMicha** [comment]
I have the same issue on nugs.net 

video link

https://securestream01.akamaized.net/nas5/pj/180813.hls/pj180813_hls_4200.m3u8?hdntl=exp=1640297776~acl=/*~data=hdntl~hmac=80836a0cedfcfe7783eb105aa63934d45a230100dc4c09e80d35fdd08bb0c82d

video is not downloading at all and after some time I receive a notification that hls downloader has crashed

tried edge and firefox

**Comment by elmanorc** [comment]
@israelmartins1 I managed to download several videos on hotmart. But the bug you said exists in deed. You can try clean the cache and try again. Sometimes it works. Se não conseguir, envia e-mail para mim (elmano@gmail.com)
